.pre-commit-config.yaml

```diff
@@ -1,40 +1,42 @@
 exclude: ^docs/
 
 repos:
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.5.0
-  hooks:
-  - id: check-yaml
-  - id: check-case-conflict
-  - id: check-json
-  - id: check-merge-conflict
-  - id: check-symlinks
-  - id: check-toml
-  - id: end-of-file-fixer
-    exclude: ^(frontend/|.*\.js$)
-  - id: requirements-txt-fixer
-  - id: detect-private-key
-- repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.3.5
-  hooks:
-    - id: ruff
-      args: [
-        --fix,
-        --select=I, # only run imports sorting
-        --exit-non-zero-on-fix,
-      ]
-    - id: ruff
-      args: [
-        --fix,
-        --exit-non-zero-on-fix,
-      ]
-    - id: ruff-format
-- repo: https://github.com/Yelp/detect-secrets
-  rev: v1.5.0
-  hooks:
-  - id: detect-secrets
-    exclude: |
-        (?x)^(
-          package.lock.json|
-          .*\.js
-        )$
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: check-yaml
+      - id: check-case-conflict
+      - id: check-json
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: check-toml
+      - id: end-of-file-fixer
+        exclude: ^(frontend/|.*\.js$)
+      - id: requirements-txt-fixer
+      - id: detect-private-key
+  - repo: https://github.com/astral-sh/ruff-pre-commit
```

Comment:
I think you have added unnecessary indentation in the existing pre-commit hooks 🤔

Reply:
actually  , it was done by prettier itself